src/fragments/lib/push-notifications/android/record_notifications/10_handle_notification_received.mdx

```diff
@@ -0,0 +1,104 @@
+<Callout>
+
+These APIs do not need to be called unless you choose to not use our service extension or you want to support multiple push providers.
+
+</Callout>
+
+## Handle Notification Received
+
+You can display the notification in the system tray if the app is in a background or killed state by calling `Amplify.Notifications.Push.handleNotificationReceived()` from the Messaging Service class. It also calls `recordNotificationReceived` irrespective of the app state.
```

Comment:
This isn't super clear on why I would use it. Why do we expose it if we only really support Pinpoint?

Reply:
I'm confused by this question. Are you asking why this is part of the Android notifications API or are you suggesting the explanation here doesn't make sense to customers?

Reply:
I think I left this comment on the wrong line - I will resolve this conversation and leave a comment in the correct spot.